### DISCUSSION TITLE
Help me save my Mindstorms 51515

### DISCUSSION THREAD

**Discussion by OneAndOnlyEcho** [pybricks/support#1337] [Q&A] [answered]
How would one upload the firmware of a spike prime hub to a mindstorms 51515?

**Comment by laurensvalk** [comment, marked as answer]
The SPIKE V2 firmware can be installed via [Pybricks](https://beta.pybricks.com/). The V3 firmware is not available/redistributable.

> Help me save my Mindstorms 51515

But if this is your goal, consider giving Pybricks a try :smile: 

We have block coding and Python coding now, and support _all_ hubs. You can even attach other motors and sensors or make multiple hubs work together.
